src/pages/volunteer/index.tsx

```diff
@@ -1,6 +1,57 @@
+import Card from '@Component/Card/Card';
+
+const Contenido_Cards = [
```

Comment:
I think all of this should be in our DATA const. ONGs will face problems if they try to modify each file with hardcoded Cards. Think in the product we're developing.